src/test/java/com/epam/tc/hw5/steps/AssertStep.java

```diff
@@ -0,0 +1,141 @@
+package com.epam.tc.hw5.steps;
+
+import static com.epam.tc.hw5.MainPage.HOME_PAGE;
+import static com.epam.tc.hw5.UserTablePage.USER_TABLE_TAB;
+
+import com.epam.tc.hw5.DifferentElementsPage;
+import com.epam.tc.hw5.DriverSetup;
+import com.epam.tc.hw5.MainPage;
+import com.epam.tc.hw5.UserTablePage;
+import io.cucumber.datatable.DataTable;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebElement;
+
+public class AssertStep extends DriverSetup {
+
+    SoftAssertions softAssert = new SoftAssertions();
+    MainPage mainPage = new MainPage(driver);
+    DifferentElementsPage differentElementsPage = new DifferentElementsPage(driver);
+    UserTablePage userTablePage = new UserTablePage(driver);
+
+    @Then("Browser title should be 'Home Page'")
+    public void assertUserTablePageIsOpened() {
```

Comment:
What do you check? What page should be opened?

Reply:
In this method, the opening of the Home page is checked.
Updated:
**_assertHomePageIsOpened_** () 
**_assertUserTableIsOpened_** ()